src/community/geogig/pom.xml

```diff
@@ -15,20 +15,21 @@
   <name>GeoGig GeoServer integration</name>
 
   <properties>
-    <geogig.version>1.3-SNAPSHOT</geogig.version>
+    <geogig.version>2.0-SNAPSHOT</geogig.version>
```

Comment:
If you look at my latest comments at this PR: https://github.com/locationtech/geogig/pull/476
you'll see that I highly discouraged upgrading to 2.0-SNAPSHOT due to the number of API changes current and coming. 1.4.x is the stable branch and master the playground. That said, suit yourself. Beware on 2.0.x the current webapi is about to be deleted for ugly, inefficient, insecure, and generally awful.

Reply:
That said it looks like I've lost admin access to geogig's Jenkins, using travis only until I can fix that with Locationtech. If you have access might want to set up a 1.4.x build job to it gets deployed.